src/app/static/src/app/components/plots/choropleth/Choropleth.vue

```diff
@@ -146,14 +158,24 @@
             },
             featureIndicators() {
                 const selectedAreaIdSet = flattenToIdSet(this.selectedAreaFilterOptions.map(o => o.id), this.flattenedAreas);
+
+                let customMin = null;
+                let customMax = null;
+                if (this.indicatorColourScale.type == ColourScaleType.Custom) {
```

Comment:
I'm moving this logic into the utils for adding dynamic range - let utils rather than choro work out what it the actual min and mac for a given trio of colour scale, metadata and data.